packages/components/view-switcher/src/view-switcher.stories.tsx

```diff
@@ -0,0 +1,144 @@
+import type { ComponentProps } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import ViewSwitcher from '.';
+import type { TViewSwitcherButtonProps } from './view-switcher-button';
+import { iconArgType } from '@/storybook-helpers';
+
+const BUTTONS_COUNT = 4;
+const DEFAULT_BUTTON_ICON = 'WorldIcon';
+
+type ViewSwitcherPropsAndCustomArgs = ComponentProps<
+  typeof ViewSwitcher.Group
+> & {
+  iconViewSwitcherButton1: string;
+  iconViewSwitcherButton2: string;
+  iconViewSwitcherButton3: string;
+  iconViewSwitcherButton4: string;
+  iconOnlyButton1: boolean;
+  iconOnlyButton2: boolean;
+  iconOnlyButton3: boolean;
+  iconOnlyButton4: boolean;
+  isDisabledButton1: boolean;
+  isDisabledButton2: boolean;
+  isDisabledButton3: boolean;
+  isDisabledButton4: boolean;
+};
+
+const meta = {
+  title: 'Components/ViewSwitcher',
+  component: ViewSwitcher.Group,
+  tags: ['autodocs'],
+} satisfies Meta<ViewSwitcherPropsAndCustomArgs>;
+
+export default meta;
+
+type Story = StoryObj<ViewSwitcherPropsAndCustomArgs>;
+
+export const Default: Story = {
+  args: {
+    isCondensed: false,
+    defaultSelected: 'Button 3',
+    iconViewSwitcherButton1: DEFAULT_BUTTON_ICON,
+    iconOnlyButton1: false,
+    isDisabledButton1: false,
+    iconViewSwitcherButton2: DEFAULT_BUTTON_ICON,
+    iconOnlyButton2: false,
+    isDisabledButton2: false,
+    iconViewSwitcherButton3: DEFAULT_BUTTON_ICON,
+    iconOnlyButton3: false,
+    isDisabledButton3: false,
+    iconViewSwitcherButton4: DEFAULT_BUTTON_ICON,
+    iconOnlyButton4: false,
+    isDisabledButton4: false,
+  },
+  argTypes: {
+    children: {
+      table: {
+        disable: true,
+      },
+    },
+    defaultSelected: {
+      table: {
+        disable: true,
+      },
+    },
```

Comment:
These two are repeated in both stories so we might want to move them to the meta definition.

Reply:
We may also consider using the [compact alternative](https://storybook.js.org/docs/essentials/controls#filtering-controls):
```jsx
parameters: {
  controls: { exclude: ['children', 'defaultSelected'] },
},
```

Reply:
Do you finally prefer to keep it like this over the compact alternative?

Reply:
Thanks, not really, I wanted to clean it up later. Anyways, I moved the implementation of `hideControls` from the other PR